base_external_dbsource_mysql/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright <2011> <Daniel Reis, Maxime Chambreuil, Savoir-faire Linux>
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    'name': 'External Database Source - MySQL',
+    'version': '12.0.1.0.0',
+    'category': 'Tools',
+    'author': "Daniel Reis, "
+              "LasLabs, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/server-backend',
+    'license': 'LGPL-3',
+    'depends': [
+        'base_external_dbsource_sqlite',
```

Comment:
Why this weird dependency?

Reply:
Because in here: 
``` python
@api.multi
    def execute_mysql(self, sqlquery, sqlparams, metadata):
        return self._execute_sqlalchemy(sqlquery, sqlparams, metadata)
```
the method `_execute_sqlalchemy` is defined in the _base_external_dbsource_sqlite_ module

Reply:
Shouldn't it be that method in base module?

Reply:
Yes, I think you're right, it should be moved to the base module